pkg/adapter/cronjobevents/adapter_test.go

```diff
@@ -26,8 +26,15 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"knative.dev/pkg/source"
 )
 
+type mockReporter struct{}
```

Comment:
can we add the eventCount mocking as in here? 
https://github.com/knative/eventing/blob/c2e66d122276763d505857b34794747e63fbc7f9/pkg/adapter/apiserver/adapter_test.go#L37
to validate that we are calling the method?

Reply:
sure

Reply:
thanks!